municipal-services/ws-calculator/src/main/java/org/egov/wscalculation/service/EstimationService.java

```diff
@@ -274,8 +274,9 @@ private List<BillingSlab> getSlabsFiltered(WaterConnection waterConnection, List
 				WaterConnectionRequest.builder().waterConnection(waterConnection).requestInfo(requestInfo).build());
 		// get billing Slab
 		log.debug(" the slabs count : " + billingSlabs.size());
-		final String buildingType = (property.getUsageCategory() != null) ? property.getUsageCategory().split("\\.")[0]
-				: "";
+		//final String buildingType = (property.getUsageCategory() != null) ? property.getUsageCategory().split("\\.")[0]: "";
+		//TODO:FIX ME : here before we passing buildingtype as UsageCategory from proerty response . but while creating property usagecategoty is residentialeven when we are creating commercial connection. Hennce we have change this to proertytype
```

Comment:
@debasishchakraborty-egovt Please explain what to be fixed here?

Reply:
This is to add the commercial rating before it was picking up residential

Reply:
@rahuldevgarg  this can go now with this. later we can modify these FIX ME part.